src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -116,33 +120,27 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
         if (result === 'Delete') {
           this.draftService.deleteDraft(this.caseDetails.case_id)
             .subscribe(_ => {
-              return this.router.navigate(['list/case'])
-                .then(() => {
-                  this.alertService.setPreserveAlerts(true);
```

Comment:
We have to make sure we alert appropriately.

Reply:
done on the subscriber side